Source/x11/XGServerEvent.m

```diff
@@ -472,10 +472,10 @@ - (void) processEvent: (XEvent *) event
             /*
              * We must hand over control of our icon/miniwindow
              * to Window Maker.
-                 */
+             */
             if ((cWin->win_attrs.window_style
-              & (NSMiniWindowMask | NSIconWindowMask)) != 0
-              && eventType == NSLeftMouseDown && clickCount == 1)
+                 & (NSMiniWindowMask | NSIconWindowMask)) != 0
```

Comment:
Something seems to be wrong with the indentation of these two lines. Could you please check?

Reply:
The indentation is correct. The line 477 aligned to the inner expression (`c` character of `cWin`). 
Line 478 aligned to the brackets of the if () statement. This is how emacs make an indentation of C code.